### PR TITLE
[ISSUE #3208] Removed ContextMenu usage

### LINKED ISSUE #3208: Migrate off MenuContext
### User Story

As a developer, I want to use patterns defined by design team

### Description

*Type*: Feature

`menu-context` have been deprecated in favor of `list-selection`. Make relevant changes.

Note: `MenuContext` is deprecated and triggers some noisy messages.

### PR DESCRIPTION
**Pull request by jeluard**
fixes #3208
edit: also #3244 
### Summary:

Moved away from ContextMenu usage

### Steps to test:
- Open Status
- Make general sanity check tests
- Validate all ContextMenu have been migrated to popup lists (e.g. in group chat settings - user profile, when showing profile keys)

status: ready

### CODE REVIEW

**Review by goranjovic — APPROVED** [review]

**Review by alwx — APPROVED** [review]
👍

**Comment by goranjovic** [comment]
^^ The same codeclimate positive.

File: src/status_im/ui/components/context_menu.cljs [hunk 1]
```diff
@@ -1,55 +0,0 @@
-(ns status-im.ui.components.context-menu
```
Comment (flexsurfer) [inline src/status_im/ui/components/context_menu.cljs:1]: my heart is bleeding, why it's so hard to watch, how someone removes code you so diligently wrote

Comment (jeluard) [reply src/status_im/ui/components/context_menu.cljs:1]: Don't cry! Ask @goranjovic , you get use to it :D

Comment (jeluard) [reply src/status_im/ui/components/context_menu.cljs:1]: Removing code is my favorite thing

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/components/context_menu.cljs (hunk 1 above)
Comment (goranjovic) [reply src/status_im/ui/components/context_menu.cljs:1]: Haha, true. Hey, I like deleting code too, just not when it's barely one month old.

**Review by yenda — APPROVED** [review]

**Comment by goranjovic** [comment]
@jeluard Please resolve the conflicts

**Comment by jeluard** [comment]
@goranjovic Done

**Comment by asemiankevich** [comment]
Branch: PR-3260
Android: https://i.diawi.com/GzgafF
iOS: https://i.diawi.com/qzUR6q

`java.lang.RuntimeException: Error calling RCTDeviceEventEmitter.emit` on android when starting the build.. iOS build does not start at all

**Comment by asemiankevich** [comment]
ARTIFACT iOS: https://i.diawi.com/zKiaEN
ARTIFACT Android: https://i.diawi.com/AS8ygH

**Pull request merged**